From d2c07b09066206edc5cabf88d164c68026e2503c Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Wed, 15 Aug 2012 10:27:54 +0800
Subject: [PATCH 042/281] ENGR00220439 imx6: not remount as ro when error ,
 panic when meet error.

panic when /data  partition meet error, not remount as ro,

the remount as ro really don't help, user may lost data after become read only
and we don't easily find I/O error happens.

Signed-off-by Zhang Jiejing <jiejing.zhang@freescale.com>
---
 imx6/sabresd/init.rc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index c81de33..b34a10f 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -99,6 +99,7 @@ on fs
 # mount ext4 partitions
     mount ext4 /dev/block/mmcblk0p5 /system
     mount ext4 /dev/block/mmcblk0p5 /system ro remount
-    mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc
+    mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc 
+errors=panic
     mount ext4 /dev/block/mmcblk0p6 /cache nosuid nodev
     mount ext4 /dev/block/mmcblk0p8 /vendor nosuid nodev
-- 
1.8.0

